bin/dfx-ckbtc-import

```diff
@@ -23,6 +23,9 @@ clap.define long=candid_dir desc="Where to store the imported candid files" vari
 # Source the output file ----------------------------------------------------------
 source "$(clap.build)"
 
+# NOTE: Fix the commit for ckbtc wasms to avoid installation problem. Remove this when it is no longer needed.
+DFX_IC_COMMIT=bcbccf79c89a3e81b1a38d8233f8f81d1af1b245
```

Comment:
We have the `DFX_IC_COMMIT` pinned here:
https://github.com/dfinity/snsdemo/blob/f7b266c557d20a2de99e7184ae06fb893cb06352/bin/versions.bash#L5

We can rely on that for a short while.